specification/schemas/Shipment.json

```diff
@@ -118,6 +118,12 @@
                 "$ref": "#/components/schemas/ShipmentItem"
               }
             },
+            "is_consolidation": {
+              "readOnly": true,
```

Comment:
We need to be able to set this via the backoffice, so it should not be read-only.